### ISSUE TITLE
issue: Antivirus (Avast) thinks that jan.ai main page contains malware...

### ISSUE THREAD

**Issue by SimplyProgrammer** [janhq/jan#3974]
### Jan version

0.5.7

### Describe the Bug

Since around the time that Jan 0.5.7 has been around. My antivirus Avast, displays URL:Mal threat immediately after opening the [Jans main page](https://jan.ai/)
The URL that it displays, nitter.vxempire.xyz, is kind a sus to be honest but I still think that it is a false positive. Nevertheless, I can tell you that it was not happening +- 1 month before. And it's probably not exactly something you want to see right after opening the page.,

### Steps to Reproduce

1. Have Avast installed.
2. Go to https://jan.ai/
3. Enjoy...

### Screenshots / Logs

![image](https://github.com/user-attachments/assets/f694ddf1-103e-4944-b26b-28552a96eae9)

### What is your OS?

- [ ] MacOS
- [x] Windows
- [ ] Linux

**Comment by imtuyethan** [comment]
Thanks for reporting this @SimplyProgrammer! We'll take a look!

**Comment by hiento09** [comment]
Cannot reproduce with the Avast extension on Chrome.
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/87db6ce6-efc5-4b43-bd10-375f76aeeb20">

I don't see any request to the URL that @SimplyProgrammer mentioned in the picture.
<img width="1509" alt="image" src="https://github.com/user-attachments/assets/204a4748-6dcb-4da5-a528-12ccd97ebd3f">

**Comment by imtuyethan** [comment]
@SimplyProgrammer We can't reproduce this, would be great if you share with us what version/plan of Avast are you using?

**Comment by louis-jan** [comment]
@SimplyProgrammer, we want to reproduce this issue. Could you tell us if you're using the Avast Pro version?

**Comment by SimplyProgrammer** [comment]
Hello, sorry for not responding...

Yes, here is the full description of my avast version:
![image](https://github.com/user-attachments/assets/fc40520a-974b-4d08-a38d-3bc75c0c05c1)

I am not using any paid version of Avast. As far as I know, I am not using any Avast chrome extension either...

**Comment by imtuyethan** [comment]
@SimplyProgrammer Hmm that is weird since we cannot reproduce this issue on our end. Quick check, can you still reproduce this issue now? Thank you for responding btw.

**Comment by urmauur** [comment]
> @SimplyProgrammer Hmm that is weird since we cannot reproduce this issue on our end. Quick check, can you still reproduce this issue now? Thank you for responding btw.

![image](https://github.com/user-attachments/assets/dd9410d1-ee6d-4f06-b3b4-351de96903c2)

Hi @SimplyProgrammer , if u still got this issue, could you please help us by recording it with the network developer tools open? It might help us with debugging

**Comment by SimplyProgrammer** [comment]
I have tried it again, and now it seems to be far less consistent... It happened to be once (today), after that I tried to refresh the page like 6 times and nothing... So I do not know.
Talking about the recording, I am not sore If I understand what you mean by that... Like the recording of the scene with the page loading? Or the "recording" output of the Network tab in google devtools (which I do not know if exporting that is even a thing)? One thing, I can tell you right now, the page is making an unholy amount of requests like really fast when it is loading initially so I do not know if you would be able to tell anything from that anyway...
![image](https://github.com/user-attachments/assets/b4538f1e-acdc-47bb-9273-ea8ab928bc23)
